scene/main/window.h

```diff
@@ -301,7 +301,7 @@ class Window : public Viewport {
 	bool is_maximize_allowed() const;
 
 	void request_attention();
-	void move_to_foreground();
+	_FORCE_INLINE_ void move_to_foreground();
```

Comment:
This won't work AFAIK, inlining requires the definition, I'd instead suggest:
```suggestion
	_FORCE_INLINE_ void move_to_foreground() { grab_focus(); }
```
And removing from the source file